.github/workflows/testing.yml

```diff
@@ -79,8 +81,7 @@ jobs:
   macos-static:
     strategy:
       matrix:
-        # Don't run on macOS 11; our setup makes clang segfault (YES).
-        os: [macos-12]
+        os: [macos-12, macos-14]
       fail-fast: false
     runs-on: ${{ matrix.os }}
```

Comment:
We actually only need a single OS for this (the same one as the release workflow), so actually, using the `matrix` is overkill.

```yaml
    runs-on: macos-14
```

(That workflow should be updated accordingly, too.)